lib/datadog/appsec/processor.rb

```diff
@@ -103,7 +103,7 @@ def load_ruleset
 
         begin
           @ruleset = case ruleset_setting
-                     when :recommended, :risky, :strict
+                     when :recommended, :strict
                        JSON.parse(Datadog::AppSec::Assets.waf_rules(ruleset_setting))
                      when String
                        JSON.parse(File.read(ruleset_setting))
```

Comment:
Done in this way, this change seems like it can cause problems when upgrading -- a customer that previously used the `risky` ruleset will start getting an error when they upgrade between dd-trace-rb 1.9.0 and a later version.

Should we perhaps do something along the lines of: when customers pick `risky`, we instead choose the closest one for them (is that `recommended` or `strict`?), and also print a warning (maybe similar [to this](https://github.com/DataDog/dd-trace-rb/blob/master/lib/datadog/core/metrics/client.rb#L188)) saying that `risky` is now a no-op.

This way the upgrade experience is always smooth.

Reply:
That is a great point @ivoanjo 

I'll ask the security folks for best closest alternative to `risky` and print a warning 😄 

Thank for the feedback

Reply:
Done https://github.com/DataDog/dd-trace-rb/pull/2598/commits/b3465c9bc477f5704e84fad76abac5effebee183

Reply:
That is a concern of me too.

@Taiki-San I think the removal of `risky` is a breaking change, which appears to run against the ruleset release SemVer 2.0 contract, WDYT?